react-charts/bullet-chart/step4.md

```diff
@@ -1,32 +1,58 @@
-Now we'll add tooltips to the bullet chart...
+Add labels to the bullet chart.
 
-## Task
+1. <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2. <strong>Locate the `<ChartBullet/>` component:</strong>
 
-2) Locate the code that looks like the following:
+It will look like this:
 
 <pre class="file">
 &lt;ChartBullet
-  comparativeWarningMeasureData={[{ name: &#39;Warning&#39;, y: 88 }]}
-  maxDomain={{y: 100}}
   primarySegmentedMeasureData={[{ name: &#39;Measure&#39;, y: 60 }]}
   qualitativeRangeData={[{ name: &#39;Range&#39;, y: 50 }, { name: &#39;Range&#39;, y: 75 }]}
+  comparativeWarningMeasureData={[{ name: &#39;Warning&#39;, y: 88 }]}
+  maxDomain={{y: 100}}
   height={150}
   width={600}
+  constraingToVisibleArea
+  labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`}
 /&gt;
 </pre>
 
-3) Add the following property to that section:
+3. <strong>Add the `padding` property to the `ChartBullet` component:</strong>
+
+This adds more padding to the left side of the chart to make room for the labels.

```

Comment:
Update this to follow area-charts:

<img width="334" alt="Screen Shot 2019-12-05 at 10 46 17 PM" src="https://user-images.githubusercontent.com/20118816/70294587-0395fc00-17b2-11ea-8360-b25b49acf2d5.png">